### DISCUSSION TITLE
http: server gave HTTP response to HTTPS client

### DISCUSSION THREAD

**Discussion by seedrick** [docker/buildx#1764] [Q&A]
I'm using docker desktop on the latest Pop!OS. I'm trying to build and push to an unsecured registry on my local network. I have both `BUILDKIT_NO_CLIENT_TOKEN=1` set in environment AND `"insecure-registries": ["<SERVER IP>:<PORT>"  ]`  set in the config in docker desktop settings. I don't want to have to set up certs and a subdomain for what's essentially an experiment, what am I missing to get this working?

**Comment by jedevc** [comment]
This looks like a duplicate of https://github.com/docker/buildx/issues/1642?

I'll close this in preference of that one, but feel free to re-open if it is different.
